blocks/social/social.js

```diff
@@ -1,7 +1,13 @@
-import { readBlockConfig, decorateIcons, fetchPlaceholders } from '../../scripts/scripts.js';
+import {
+  readBlockConfig,
+  decorateIcons,
+  fetchPlaceholders,
+  updateExternalLinks,
```

Comment:
i (finally) noticed these links weren't opening in a new tab, so i snuck in that missing piece...